Core/SqlGeneration/SqlGeneratingExpressionVisitor.cs

```diff
@@ -386,17 +386,13 @@ public virtual Expression VisitAggregation (AggregationExpression expression)
 
       return expression;
     }
-
-    public virtual Expression VisitSqlCollection (SqlCollectionExpression expression)
+  
```

Comment:
```suggestion

```

Reply:
This line separates two methods. I have no idea why the change is "delete empty line plus method declaration, add empty line" instead of just "delete method declaration", but anyway the empty line should be there.